app/src/i18n/locales/en/data.json

```diff
@@ -159,5 +159,6 @@
   "download-template": "Download template to upload data",
   "file-upload-steps": "Follow the steps in our template and upload the data to calculate your inventory without waiting for CityCatlyst's data review.",
   "add-custom": "Add custom",
-  "save-missing-scope-info": "Save or fill in missing information before proceeding."
+  "save-missing-scope-info": "Save or fill in missing information before proceeding.",
+  "disconnect-data": "Disconnect Data"
```

Comment:
Can you get the translations for our other supported languages? I can give you the German ones if that helps.

```
"save-missing-scope-info": "Speichern Sie oder geben Sie die fehlenden Informationen ein before Sie fortfahren.",
  "disconnect-data": "Datenquelle trennen"
  ```